xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/extension.vm

```diff
@@ -1037,13 +1037,15 @@ $namespace##
       #end
     </div>
     <div class="extension-header">
-      <h2 class="extension-title">
-        <span class="extension-name">#displayExtensionName($extension)</span>
-        <span class="extension-version">$escapetool.xml($extension.id.version)</span>
-      </h2>
-      #if ($extensionStatusMessage)
-        <p class="extension-status">$escapetool.xml($extensionStatusMessage)</p>
-      #end
+      <div class="extension-title-and-status">
+        <h2 class="extension-title">
+          <span class="extension-name">#displayExtensionName($extension)</span>
+          <span class="extension-version">$escapetool.xml($extension.id.version)</span>
+        </h2>
+        #if ($extensionStatusMessage)
+          <p class="extension-status">$escapetool.xml($extensionStatusMessage)</p>
+        #end
```

Comment:
I would rather move ``extension-status`` inside ``extension-title`` than introducing another wrapper.

Reply:
I agree we should use wrappers wisely. I'm trying to implement a layout where there are two elements on the left (the extension name and its version) and one on the right (the extension status). Won't we end up grouping the two first ones in a common container from which the third one is excluded? Or do you see an alternative?

Reply:
It might be possible to use flex on the ``extension-title`` and set ``flex-grow`` on the ``extension-status`` to make it grow to fill the remaining space. Then align the text to the right on the ``extension-status``. I haven't tried it though.

Reply:
Indeed, good idea, thanks. The latest commit follows this path. It also sets the `dependency-item` background image position to make sure it remains aligned at the top even when the item spans across multiple lines, just like how the `extension-title` behaves. I'm adding screenshots to the JIRA issue to clarify.

Reply:
Meantime I actually removed the `dependency-item` background image positionining rule because it does not align the icon correctly in case of large font. This might be the subject of another issue.

Reply:
In the new layout, the extension version number is not wrapped (eg "Refactoring application"), while it was when the extension name and its version were grouped in a common block:

## New layout without additional wrapper

![extension-updates-ld](https://user-images.githubusercontent.com/1022195/116433958-472b5300-a84a-11eb-809c-6b77dc33cd75.jpg)

However in the dependencies panel, the extension name and versions are wrapped, which can result in a slightly different display (eg with the "Administration Application"), this is something we may want to align visually in the scope of a different issue maybe:

![extension-dependencies-ld](https://user-images.githubusercontent.com/1022195/116435104-4fd05900-a84b-11eb-9a01-ab4b2e15e7a7.jpg)

## Previous layout where the title and the version were wrapped

![large-screen-display](https://user-images.githubusercontent.com/1022195/116435403-96be4e80-a84b-11eb-86ad-db546980e2a6.jpg)